src/components/CustomizableCalendarDay.jsx

```diff
@@ -291,45 +311,28 @@ class CustomizableCalendarDay extends React.Component {
       ariaLabel,
     } = getCalendarDaySettings(day, ariaLabelFormat, daySize, modifiers, phrases);
 
-    const defaultStyles = getStyles(defaultStylesWithHover, isHovered);
-    const outsideStyles = getStyles(outsideStylesWithHover, isHovered);
-    const todayStyles = getStyles(todayStylesWithHover, isHovered);
-    const firstDayOfWeekStyles = getStyles(firstDayOfWeekStylesWithHover, isHovered);
-    const lastDayOfWeekStyles = getStyles(lastDayOfWeekStylesWithHover, isHovered);
-    const highlightedCalendarStyles = getStyles(highlightedCalendarStylesWithHover, isHovered);
-    const blockedMinNightsStyles = getStyles(blockedMinNightsStylesWithHover, isHovered);
-    const blockedCalendarStyles = getStyles(blockedCalendarStylesWithHover, isHovered);
-    const blockedOutOfRangeStyles = getStyles(blockedOutOfRangeStylesWithHover, isHovered);
-    const hoveredSpanStyles = getStyles(hoveredSpanStylesWithHover, isHovered);
-    const selectedSpanStyles = getStyles(selectedSpanStylesWithHover, isHovered);
-    const lastInRangeStyles = getStyles(lastInRangeStylesWithHover, isHovered);
-    const selectedStartStyles = getStyles(selectedStartStylesWithHover, isHovered);
-    const selectedEndStyles = getStyles(selectedEndStylesWithHover, isHovered);
-    const selectedStyles = getStyles(selectedStylesWithHover, isHovered);
-    const afterHoveredStartStyles = getStyles(afterHoveredStartStylesWithHover, isHovered);
-
     return (
       <td
         {...css(
           styles.CalendarDay,
           useDefaultCursor && styles.CalendarDay__defaultCursor,
           daySizeStyles,
-          defaultStyles,
-          isOutsideDay && outsideStyles,
-          modifiers.has('today') && todayStyles,
-          modifiers.has('first-day-of-week') && firstDayOfWeekStyles,
-          modifiers.has('last-day-of-week') && lastDayOfWeekStyles,
-          modifiers.has('highlighted-calendar') && highlightedCalendarStyles,
-          modifiers.has('blocked-minimum-nights') && blockedMinNightsStyles,
-          modifiers.has('blocked-calendar') && blockedCalendarStyles,
-          hoveredSpan && hoveredSpanStyles,
-          modifiers.has('after-hovered-start') && afterHoveredStartStyles,
-          modifiers.has('selected-span') && selectedSpanStyles,
-          modifiers.has('last-in-range') && lastInRangeStyles,
-          selected && selectedStyles,
-          modifiers.has('selected-start') && selectedStartStyles,
-          modifiers.has('selected-end') && selectedEndStyles,
-          isOutsideRange && blockedOutOfRangeStyles,
+          getStyles(defaultStylesWithHover, isHovered),
```

Comment:
couldn't think of a good intermediate name so I just execute everything inline

Reply:
That seems reasonable to me